testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1346,3 +1369,90 @@ def server_secret
+  profiles = ["orchid", "flame", "pearl"]
+  isolinux_profiles = []
+  cobbler_profiles = []
+  profiles.each do |profile|
+    # get all profiles from Cobbler
+    result_cobbler, code = $server.run("cobbler profile list | grep -o #{profile}")
+    cobbler_profiles.push(result_cobbler) if code.zero?
+    # get all profiles from isolinux.cfg
+    result_isolinux, code = $server.run("cat #{tmp_dir}/isolinux/isolinux.cfg | grep -o #{profile} | cut -c -6 | head -n 1")
+    unless result_isolinux.empty?
+      isolinux_profiles.push(result_isolinux)
+    end
+  end
+  raise 'error during comparison of Cobbler profiles' unless cobbler_profiles == isolinux_profiles
+end
+
+When(/^I run Cobbler buildiso for distro "([^"]*)" and profile "([^"]*)"$/) do |distro, profile|
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  iso_dir = "/var/cache/cobbler"
+  _out, code = $server.run("cobbler buildiso --tempdir=#{tmp_dir} --iso #{iso_dir}/#{profile}.iso --distro=#{distro} --profile=#{profile}")
+  raise 'error in cobbler buildiso, check the Cobbler logs' if code.nonzero?
+end
+
+When(/^I run Cobbler buildiso for distro "([^"]*)" and profile "([^"]*)" without dns entries$/) do |distro, profile|
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  iso_dir = "/var/cache/cobbler"
+  _out, code = $server.run("cobbler buildiso --tempdir=#{tmp_dir} --iso #{iso_dir}/#{profile}.iso --distro=#{distro} --profile=#{profile} --exclude-dns")
+  raise 'error in cobbler buildiso, check the Cobbler logs' if code.nonzero?
+  # TODO Check for missing nameserver entry in isolinux.cfg
+  result, code = $server.run("cat #{tmp_dir}/isolinux/isolinux.cfg | grep -o nameserver")
+  # we have to fail here if the command suceeds
+  raise 'error in Cobbler buildiso, nameserver parameter found in isolinux.cfg but should not be found' if code.zero?
+end
+
+When(/^I run Cobbler buildiso "([^"]*)" for distro "([^"]*)"$/) do |param, distro|
+  # param can either be standalone or airgapped
+  # workaround to get the contents of the buildiso folder
+  step %(I run Cobbler buildiso for distro "#{distro}" and all profiles)
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  iso_dir = "/var/cache/cobbler"
+  source_dir = "/var/cache/cobbler/source_#{param}"
+  $server.run("mv #{tmp_dir} #{source_dir}")
+  $server.run("mkdir -p #{tmp_dir}")
+  _out, code = $server.run("cobbler buildiso --tempdir=#{tmp_dir} --iso #{iso_dir}/#{param}.iso --distro=#{distro} --#{param} --source=#{source_dir}")
+  raise 'error in cobbler buildiso, check the Cobbler logs' if code.nonzero?
+end
+
+And(/^I check Cobbler buildiso ISO "([^"]*)" with xorriso$/) do |name|
+  tmp_dir = "/var/cache/cobbler"
+  _out, code = $server.run("cat >#{tmp_dir}/test_image <<-EOF
+BIOS
+UEFI
+EOF")
+  _out, code = $server.run("xorriso -indev #{tmp_dir}/#{name}.iso -report_el_torito 2>/dev/null | awk '/^El Torito boot img[[:space:]]+:[[:space:]]+[0-9]+[[:space:]]+[a-zA-Z]+[[:space:]]+y/{print $7}' >>#{tmp_dir}/xorriso_#{name}")
```

Comment:
This line looks too long for me... we should make it shorter.

Reply:
I copied this over from the upstream integration test:
https://github.com/cobbler/cobbler/blob/f440affa5cb9d41e589033be5c3511f18fcac348/system-tests/tests/buildiso-%40#L59-L60. I did not found a better solution to shorten this, yet.

Reply:
Yup, that's what I was suggesting :)
I guess Rubocop don't complains just because we are disabling the Cop about it :)

Reply:
Ok. I will adjust this with another commit.